### DISCUSSION TITLE
Context Menu Not Available In DataGrid

### DISCUSSION THREAD

**Discussion by OmerFarukAkyapak** [amwx/FluentAvalonia#534] [Q&A] [answered]
I cant use context menu in data grid. It is not available for data grid  in this package. How to use it ??

![image](https://github.com/amwx/FluentAvalonia/assets/56655714/dc0130ba-86bf-4a0e-902a-ace9e00d2e37)

**Comment by timunie** [comment, marked as answer]
Try:

```xml
<DataGrid.ContextFlyout>
    <MenuFlyout>
        <MenuItem Header="Test"></MenuItem>
    </MenuFlyout>
</DataGrid.ContextFlyout>
```

**Reply by OmerFarukAkyapak** [reply]
Thanks, it done :)
